DynamicMissionGeneratorAssembly/src/MissionInputPage.cs

```diff
@@ -956,13 +963,19 @@ private string UpdateModeSettingsForMission(DMGMission mission)
 			// Make sure to do this after processing in case an error was returned anywhere.
 			if (tweakSettings != null)
 			{
+				// Merge our settings with Tweaks in case there is any new settings.
+				var mergedDictionary = MergeSettings(tweaksPath, tweakSettings);
+
 				File.Copy(tweaksPath, Path.Combine(modSettingsPath, "TweakSettings.json.bak"), true); // I've now added the override bool JUST in case.
-				File.WriteAllText(tweaksPath, JsonConvert.SerializeObject(tweakSettings, Formatting.Indented));
+				File.WriteAllText(tweaksPath, JsonConvert.SerializeObject(mergedDictionary, Formatting.Indented, new StringEnumConverter()));
```

Comment:
the type or namespace StringEnumConverter cannot be found - please resolve before I merge this